src/perf/lib/PerfServer.h

```diff
@@ -154,6 +216,20 @@ class PerfServer {
     TcpEngine Engine;
     TcpServer Server;
 
+    uint32_t DelayMicroseconds = 0;
+    SYNTHETIC_DELAY_TYPE DelayType = SYNTHETIC_DELAY_FIXED;
+    bool DelayPoolUsed = FALSE;
+    DelayWorker* DelayWorkers = nullptr;
+    uint16_t ProcCount = 0;
```

Comment:
Let's follow the convention above:
```suggestion
    uint32_t DelayMicroseconds {0};
    SYNTHETIC_DELAY_TYPE DelayType {SYNTHETIC_DELAY_FIXED};
    bool DelayPoolUsed {FALSE};
    DelayWorker* DelayWorkers {nullptr};
    uint16_t ProcCount {0};
```